package-scripts.js

```diff
@@ -24,13 +24,7 @@ module.exports = {
       )
     },
     dev: {
-      default: series(
-        'git submodule update --init --recursive --remote',
-        'npm run installWeb3',
-        'bash ./scripts/generateProxyContracts.sh',
-        'npm run moveTruffleConfigPublic',
-        'npm run moveSolcVersionOutputPublic'
-      ),
+      default: series('git submodule update --init --recursive --remote', 'npm run installWeb3'),
       Minted: {
         default: series(
           'nps test.e2e.stop',
```

Comment:
@mariano-aguero is it possible that in a clean installation, web3 is missing?

I added the line npm run installWeb3 and everything worked fine.

(same with `Dutch`)

Reply:
@fernandomg I think that the default script that includes the installation of web3 is executed first and then the dev:minted or dev:dutch script.
Anyway, I move the script to make it functional